### PR TITLE
feat(docs): added accessibility requirement

### PR DESCRIPTION
**Pull request by Spitgranger**
This PR adds the missing accessibility requirement.

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -656,7 +656,13 @@ \subsection{Understandability and Politeness Requirements}
     determine next steps.
 \end{enumerate}
 \subsection{Accessibility Requirements}
-TBD may be in constraints.
+\begin{enumerate}[{UH-AS}1.]
+  \item The application must adhere to accessibility policies set by the City Of
+    Hamilton \href{https://www.hamilton.ca/people-programs/inclusion-diversity-equity-accessibility/accessibility-services/accessibility#policies-procedures}{Linked Here}.\\
```
Comment (Spitgranger) [inline docs/SRS-Volere/SRS.tex:661]: This line is over 80, but the hyperlink won't work if its split on multiple lines

Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex:661]: is the linter not allowing the commit because of that? It let my hyperlinks through when Kyle set it up

Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex:661]: yes it does allow the hyperlinks it just gives a warning for that

Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex:661]: I'll fix the formatting

Comment (Spitgranger) [reply docs/SRS-Volere/SRS.tex]: Where is the ruleset? Everyone should have the same ruleset on the formatter to prevent these issues

Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex]: it's just the default for tex-fmt. If you wanna install do `cargo install tex-fmt`

**Review by kyle4908 — APPROVED** [review]

**Pull request merged**